app/views/admin/rys_management/_view_admin_plugins_bottom.html.erb

```diff
@@ -19,7 +19,11 @@
         <td><%= plugin.feature_records_count %></td>
 
         <td class="buttons text-right">
-          <%= link_to l(:button_edit), rys_management_edit_path(plugin.rys_id), class: 'icon icon-edit' %>
+          <% if plugin.active? %>
```

Comment:
See my comments on rys plugin and next time write a link or say it requires modification from elsewhere.